src/components/backend-ai-import-view.ts

```diff
@@ -80,22 +89,73 @@ export default class BackendAIImport extends BackendAIPage {
           margin: 10px auto;
         }
 
-        mwc-textfield#notebook-url,
-        mwc-textfield#github-repo-url {
+        mwc-textfield#notebook-url {
           width: 75%;
         }
+        mwc-textfield#github-repo-url {
+          width: 100%;
+        }
+        mwc-textfield#gitlab-repo-url {
+          width: 100%;
+        }
```

Comment:
Let's merge these blocks into:
```javascript
mwc-textfield#github-repo-url,
mwc-textfield#gitlab-repo-url {
  width: 100%;
}
```

Or, just tag the same class, such as `repo-url`, to both of the elements and:
```javascript
mwc-textfield.repo-url {
  ...
} 
```